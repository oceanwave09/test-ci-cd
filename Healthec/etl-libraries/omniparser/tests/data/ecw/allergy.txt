apuid|RecordID|PatientId|PatientAccountNo|EncounterId|EncounterDate|ProviderNPI|ItemID|AllergiesVerified|AllergenType|AllergyDescription|AllergyReactionText|AllergyStatus|AllergyID|ModifiedDate
280807|280807.17502.606277.0.4|17502|17502|606277|2016-04-28|1245324565|0|Y|Allergy|Tree and Grass Pollen|Rhinitis|Active||2016-04-28
280807|280807.17502.606277.0.5|17502|17502|606277|2016-04-28|1245324565|0|Y|Allergy|Early and Late Weeds|Rhinitis|Active||2016-04-28
280807|280807.17502.606277.0.6|17502|17502|606277|2016-04-28|1245324565|0|Y|Allergy|Ragweed |Rhinitis|Active||2016-04-28
280807|280807.17502.606277.262974.1|17502|17502|606277|2016-04-28|1245324565|262974|Y|Allergy|Augmentin|Unknown|Active|00029-6085-22|2016-04-28
280807|280807.17502.606277.273264.2|17502|17502|606277|2016-04-28|1245324565|273264|Y|Allergy|Penicillin G Potassium|Unknown|Active|00781-6136-94|2016-04-28
280807|280807.17502.606277.278365.3|17502|17502|606277|2016-04-28|1245324565|278365|Y|Allergy|Vistaril|Unknown|Active|00069-5410-66|2016-04-28
280807|280807.17502.514628.0.1|17502|17502|514628|2015-02-19|1245324565|0|Y|Allergy|Molds: A|rhinitis|Active||2015-02-19
280807|280807.17502.514628.0.2|17502|17502|514628|2015-02-19|1245324565|0|Y|Allergy|Molds:  B|rhinitis|Active||2015-02-19
280807|280807.17502.2525555.276781.1|17502|17502|2525555|2022-10-12|1417179151|276781||Allergy|Sulfur|rash|Active|00145-1024-03|2022-10-12
280807|280807.17502.2534007.0.1|17502|17502|2534007|2022-10-17|1518170364|0||Allergy|Tree Pollen Grass Pollun ana Ragweed Pollen|rhinitis|Active||2022-10-17
280807|280807.17502.2534007.276747.3|17502|17502|2534007|2022-10-17|1518170364|276747||Allergy|Sulfacetamide Sodium|hives|Active|24208-0670-04|2022-10-17
280807|280807.17502.2546108.262974.1|17502|17502|2546108|2022-10-17|1679623896|262974|Y|Allergy|Augmentin|anaphylaxis|Active|43598-0006-14|2022-10-17
280807|280807.17502.2546108.265354.2|17502|17502|2546108|2022-10-17|1679623896|265354|Y|Allergy|Cortisone Acetate|anaphylaxis|Active||2022-10-17
280807|280807.17502.2546108.281790.3|17502|17502|2546108|2022-10-17|1679623896|281790|Y|Allergy|Clozaril|anaphylaxis|Active||2022-10-17
280807|280807.17502.2544650.0.1|17502|17502|2544650|2022-10-17|1679623896|0|Y|Allergy|Early weeds, late weeds, and ragweed|rhinitis|Active||2022-10-17
280807|280807.17502.2544650.0.2|17502|17502|2544650|2022-10-17|1679623896|0|Y|Allergy|Tree and grass pollen|rhinitis|Active||2022-10-17
280807|280807.17502.2544650.0.3|17502|17502|2544650|2022-10-17|1679623896|0|Y|Allergy|Dust mites|rhinitis|Active||2022-10-17
280807|280807.17502.2517560.0.1|17502|17502|2517560|2022-10-17|1679623896|0|Y|Allergy|Tree and Grass Pollen|rhinitis|Active||2022-10-17
280807|280807.17502.2517560.0.2|17502|17502|2517560|2022-10-17|1679623896|0|Y|Allergy|Early and Late Weeds|rhinitis|Active||2022-10-17
280807|280807.17502.2517560.0.3|17502|17502|2517560|2022-10-17|1679623896|0|Y|Allergy|Ragweed Pollen|rhinitis|Active||2022-10-17
280807|280807.17502.2517560.0.4|17502|17502|2517560|2022-10-17|1679623896|0|Y|Allergy|Mold Spores|rhinitis|Active||2022-10-17
280807|280807.17502.2542571.0.1|17502|17502|2542571|2022-10-17|1679623896|0|Y|Allergy|BEANS/CABBAGE|rash|Active||2022-10-17
280807|280807.17502.2542571.278132.3|17502|17502|2542571|2022-10-17|1679623896|278132|Y|Allergy|Valium|Unknown|Active|35356-0006-00|2022-10-17
280807|280807.17502.2546963.444830.1|17502|17502|2546963|2022-10-17|1396385019|444830|Y|Allergy|Dust Mite Mixed Allergen Ext|rash|Active|00268-7002-10|2022-10-17
280807|280807.17502.2552645.0.2|17502|17502|2552645|2022-10-17|1679623896|0|Y|Allergy|seasonal|Unknown|Active||2022-10-17
280807|280807.17502.2552645.273269.1|17502|17502|2552645|2022-10-17|1679623896|273269|Y|Allergy|Penicillin V Potassium|rash|Active|00093-1172-01|2022-10-17
280807|280807.17502.2550040.0.1|17502|17502|2550040|2022-10-17|1841223997|0|Y|Allergy|seasonal||Active||2022-10-17
280807|280807.17502.2552458.266035.1|17502|17502|2552458|2022-10-17|1841223997|266035|Y|Allergy|Demerol|rash|Active|00409-1180-59|2022-10-17
280807|280807.17502.2506965.273261.1|17502|17502|2506965|2022-10-17|1235225871|273261|Y|Allergy|Penicillamine|(10/01/2013)|Active|00254-2000-01|2022-10-17
