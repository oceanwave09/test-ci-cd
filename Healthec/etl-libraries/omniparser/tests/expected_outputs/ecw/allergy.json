[
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Tree and Grass Pollen",
		"allergy_reaction_text": "Rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2016-04-28",
		"encounter_id": "606277",
		"item_id": "0",
		"modified_date": "2016-04-28",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "589e77a0e94f5a1e368c53a2646e4956",
		"record_id": "280807.17502.606277.0.4",
		"row_id": "90d3b837-4a8b-43d5-bf08-228438f76f10"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Early and Late Weeds",
		"allergy_reaction_text": "Rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2016-04-28",
		"encounter_id": "606277",
		"item_id": "0",
		"modified_date": "2016-04-28",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "589e77a0e94f5a1e368c53a2646e4956",
		"record_id": "280807.17502.606277.0.5",
		"row_id": "abef465e-c3c2-435a-915f-89e3227640f0"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Ragweed",
		"allergy_reaction_text": "Rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2016-04-28",
		"encounter_id": "606277",
		"item_id": "0",
		"modified_date": "2016-04-28",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "589e77a0e94f5a1e368c53a2646e4956",
		"record_id": "280807.17502.606277.0.6",
		"row_id": "8e04da9e-88a8-447d-83f7-40191b93b518"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Augmentin",
		"allergy_id": "00029-6085-22",
		"allergy_reaction_text": "Unknown",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2016-04-28",
		"encounter_id": "606277",
		"item_id": "262974",
		"modified_date": "2016-04-28",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "589e77a0e94f5a1e368c53a2646e4956",
		"record_id": "280807.17502.606277.262974.1",
		"row_id": "0256a304-e641-4169-8afd-df719c65948e"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Penicillin G Potassium",
		"allergy_id": "00781-6136-94",
		"allergy_reaction_text": "Unknown",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2016-04-28",
		"encounter_id": "606277",
		"item_id": "273264",
		"modified_date": "2016-04-28",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "589e77a0e94f5a1e368c53a2646e4956",
		"record_id": "280807.17502.606277.273264.2",
		"row_id": "0c8635ce-4559-48bd-8b24-a7904f7a8f9a"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Vistaril",
		"allergy_id": "00069-5410-66",
		"allergy_reaction_text": "Unknown",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2016-04-28",
		"encounter_id": "606277",
		"item_id": "278365",
		"modified_date": "2016-04-28",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "589e77a0e94f5a1e368c53a2646e4956",
		"record_id": "280807.17502.606277.278365.3",
		"row_id": "873c8ce1-5a0c-4eb7-b7f9-60d29006deb1"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Molds: A",
		"allergy_reaction_text": "rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2015-02-19",
		"encounter_id": "514628",
		"item_id": "0",
		"modified_date": "2015-02-19",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "589e77a0e94f5a1e368c53a2646e4956",
		"record_id": "280807.17502.514628.0.1",
		"row_id": "0a9e4873-6310-429b-b143-9ec952f0d770"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Molds:  B",
		"allergy_reaction_text": "rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2015-02-19",
		"encounter_id": "514628",
		"item_id": "0",
		"modified_date": "2015-02-19",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "589e77a0e94f5a1e368c53a2646e4956",
		"record_id": "280807.17502.514628.0.2",
		"row_id": "beba73f2-f884-42ed-9d6c-9d66f52a1f07"
	},
	{
		"allergen_type": "Allergy",
		"allergy_description": "Sulfur",
		"allergy_id": "00145-1024-03",
		"allergy_reaction_text": "rash",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-12",
		"encounter_id": "2525555",
		"item_id": "276781",
		"modified_date": "2022-10-12",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cc8473289580a14bfaa0423d40f320e6",
		"record_id": "280807.17502.2525555.276781.1",
		"row_id": "9c65c062-f3d8-4c73-a615-bc8b270dcc6e"
	},
	{
		"allergen_type": "Allergy",
		"allergy_description": "Tree Pollen Grass Pollun ana Ragweed Pollen",
		"allergy_reaction_text": "rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2534007",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cf27ba2585b8d9acf4e4e718e6fa1723",
		"record_id": "280807.17502.2534007.0.1",
		"row_id": "c4e5aa49-f1cd-4b95-8d10-2b8b8c8c085f"
	},
	{
		"allergen_type": "Allergy",
		"allergy_description": "Sulfacetamide Sodium",
		"allergy_id": "24208-0670-04",
		"allergy_reaction_text": "hives",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2534007",
		"item_id": "276747",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cf27ba2585b8d9acf4e4e718e6fa1723",
		"record_id": "280807.17502.2534007.276747.3",
		"row_id": "39ccbd17-5869-42f5-beb8-2498631260b5"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Augmentin",
		"allergy_id": "43598-0006-14",
		"allergy_reaction_text": "anaphylaxis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2546108",
		"item_id": "262974",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2546108.262974.1",
		"row_id": "8ba0e58b-b9d1-4412-b739-f02a99071450"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Cortisone Acetate",
		"allergy_reaction_text": "anaphylaxis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2546108",
		"item_id": "265354",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2546108.265354.2",
		"row_id": "c9e83c3f-7588-4396-8fdb-d8b22e6c0dab"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Clozaril",
		"allergy_reaction_text": "anaphylaxis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2546108",
		"item_id": "281790",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2546108.281790.3",
		"row_id": "59d7d257-aa33-4c49-9ed7-40654a3e6331"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Early weeds, late weeds, and ragweed",
		"allergy_reaction_text": "rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2544650",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2544650.0.1",
		"row_id": "e51b1a9e-4e9a-4d62-9d92-2ed3029496d6"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Tree and grass pollen",
		"allergy_reaction_text": "rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2544650",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2544650.0.2",
		"row_id": "62a2779c-7a1d-4834-ae2d-7d679368c225"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Dust mites",
		"allergy_reaction_text": "rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2544650",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2544650.0.3",
		"row_id": "aa5bd390-62ff-4006-9ecd-bacefb167613"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Tree and Grass Pollen",
		"allergy_reaction_text": "rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2517560",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2517560.0.1",
		"row_id": "ee2746f9-2154-4981-9c4f-9d6391e8ac52"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Early and Late Weeds",
		"allergy_reaction_text": "rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2517560",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2517560.0.2",
		"row_id": "29ac2853-6e6d-45c1-a6a5-af10762bd948"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Ragweed Pollen",
		"allergy_reaction_text": "rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2517560",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2517560.0.3",
		"row_id": "48953d8c-138b-4502-9edf-31fedd17aacd"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Mold Spores",
		"allergy_reaction_text": "rhinitis",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2517560",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2517560.0.4",
		"row_id": "5fc42f6d-5089-4237-b2f6-6453d0ed1c8a"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "BEANS/CABBAGE",
		"allergy_reaction_text": "rash",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2542571",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2542571.0.1",
		"row_id": "9f8b03f0-e649-45fa-9481-bd0b028807ae"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Valium",
		"allergy_id": "35356-0006-00",
		"allergy_reaction_text": "Unknown",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2542571",
		"item_id": "278132",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2542571.278132.3",
		"row_id": "d1815af3-b95d-4c4c-969c-b105d3a9825d"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Dust Mite Mixed Allergen Ext",
		"allergy_id": "00268-7002-10",
		"allergy_reaction_text": "rash",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2546963",
		"item_id": "444830",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"record_id": "280807.17502.2546963.444830.1",
		"row_id": "3c0e0fe1-23d9-4653-9613-ca0dce31cff3"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "seasonal",
		"allergy_reaction_text": "Unknown",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2552645",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2552645.0.2",
		"row_id": "d52dadea-9002-4ac9-8548-28b88e01c6d1"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Penicillin V Potassium",
		"allergy_id": "00093-1172-01",
		"allergy_reaction_text": "rash",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2552645",
		"item_id": "273269",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"record_id": "280807.17502.2552645.273269.1",
		"row_id": "83b4600a-b786-4d56-9f34-ce09bb8ac5a5"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "seasonal",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2550040",
		"item_id": "0",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"record_id": "280807.17502.2550040.0.1",
		"row_id": "d2b20a02-e5e4-4d35-9b07-65c4b2b5c50f"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Demerol",
		"allergy_id": "00409-1180-59",
		"allergy_reaction_text": "rash",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2552458",
		"item_id": "266035",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"record_id": "280807.17502.2552458.266035.1",
		"row_id": "27cc504a-fad6-4ad1-a633-3de359247f49"
	},
	{
		"allergen_type": "Allergy",
		"allergies_verified": "Y",
		"allergy_description": "Penicillamine",
		"allergy_id": "00254-2000-01",
		"allergy_reaction_text": "(10/01/2013)",
		"allergy_status": "Active",
		"apuid": "280807",
		"encounter_date": "2022-10-17",
		"encounter_id": "2506965",
		"item_id": "273261",
		"modified_date": "2022-10-17",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "3a0d3850d7f4164bf6c5df8ca687c62a",
		"record_id": "280807.17502.2506965.273261.1",
		"row_id": "a6892139-fa5e-4e76-86e9-c36eb4b9264f"
	}
]
